easybuild/easyconfigs/r/RAxML/RAxML-7.7.5-ictce-5.3.0-mt-sse3.eb

```diff
@@ -15,7 +15,8 @@ source_urls = ['https://github.com/stamatak/standard-RAxML/archive/']
 
 skipsteps = ['configure', 'install']
 
-buildopts = '-f Makefile.SSE3.PTHREADS.gcc CC="$CC" && mkdir -p %(installdir)s/bin && cp raxmlHPC-PTHREADS-SSE3 %(installdir)s/bin'
+buildopts = '-f Makefile.SSE3.PTHREADS.gcc CC="$CC" && '
+buildopts += "mkdir -p %(installdir)s/bin && cp raxmlHPC-PTHREADS-SSE3 %(installdir)s/bin"
```

Comment:
same here: contains spaces, hence `"`

line above requires use of `"..."` to ensure bash expansion of `$CC`, so there I use single quotes for the string